src/wincameranode.cpp

```diff
@@ -17,13 +29,13 @@ int main(int argc, char** argv)
     int32_t queueSize = PUBLISHER_QUEUE_SIZE;
     winrt::com_ptr< ros_win_camera::WindowsMFCapture> camera, camera1;
     std::string frame_id("camera");
-    privateNode.param("frame_rate", frameRate, 30.0f);
+    privateNode.param("frame_rate", frameRate, DEFAULT_FRAMERATE);
     privateNode.param("pub_queue_size", queueSize, PUBLISHER_QUEUE_SIZE);
     std::mutex waitForFinish;
-    int32_t Width(640), Height(480);
+    int32_t Width(DEFAULT_WIDTH), Height(DEFAULT_HEIGHT);

```

Comment:
>int32_t [](start = 4, length = 7)

why use a signed int

Reply:
Rosnode getParam does not have support for unsigned int.